catprinter/img.py

```diff
@@ -23,15 +23,42 @@ def adjust_pixel(y, x, delta):
             new_val = 255 if img[y][x] > 127 else 0
             err = img[y][x] - new_val
             img[y][x] = new_val
-            adjust_pixel(y, x + 1, err * 7/16)
+            adjust_pixel(y, x + 1, err * 1/16)
```

Comment:
I'm not 100% sure, but I think this was supposed to really be `7` (at least according to [wiki](https://en.wikipedia.org/wiki/Floyd%e2%80%93Steinberg_dithering)).

Reply:
Oh, that might be a copy/paste error on my part. Good eye! I didn't intend to change the Steinberg dithering at all. Thanks!